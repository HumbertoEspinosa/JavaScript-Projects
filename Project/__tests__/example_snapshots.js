

//para actualizar el snapshots con otro actual que este usando npm t -- -u
const client={
    name:'Martha',
    balance: 4000,
    type:'premium'
}

describe('Testeando al cliente',()=>{
    test('Es Martha?',()=>{
        expect(client).toMatchSnapshot()
    })

})