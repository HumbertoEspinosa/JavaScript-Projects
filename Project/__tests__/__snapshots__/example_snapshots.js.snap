// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testeando al cliente Es Martha? 1`] = `
Object {
  "balance": 4000,
  "name": "Martha",
  "type": "premium",
}
`;
